dashboard/db/migrate/20160809170542_add_view_answers_unlocked_at_to_user_levels.rb

```diff
@@ -0,0 +1,6 @@
+class AddViewAnswersUnlockedAtToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :view_answers, :boolean
```

Comment:
I currently treat false and NULL to be equivalent. Would it be better to say it's non-nullable and have a default value of false?